src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -85,7 +85,7 @@ internal void HandleOnRecoveryError(object sender, RecoveryErrorEventArgs e)
 
         public DocumentDatabase(string name, RavenConfiguration configuration, ServerStore serverStore)
         {
-            Scripts = new ScriptRunnerCache(this);
+            Scripts = new ScriptRunnerCache(this, Configuration);
```

Comment:
I think that this should be local variable, the 'Configuration' is set few lines lower.

Reply:
The real issue here is that Scripts should be created last, after Configuration property is set

Reply:
No need to pass the configuration separately, just move the initialization after setting the Configuration property

Reply:
Done

Reply:
We pass the configuration separately for the case that database is null and the configuration is the serverStore's one.